Enable building a pure soft-float compiler without the need for a software
floating point library.

diff -dur gcc-4.0.4.orig/gcc/config.gcc gcc-4.0.4/gcc/config.gcc
--- gcc-4.0.4.orig/gcc/config.gcc	2007-02-02 19:12:28.000000000 +0100
+++ gcc-4.0.4/gcc/config.gcc	2007-02-02 19:12:07.000000000 +0100
@@ -677,7 +677,7 @@
 		tm_defines="TARGET_BIG_ENDIAN_DEFAULT=1 $tm_defines"
 		;;
 	esac
-	tmake_file="${tmake_file} arm/t-arm arm/t-linux"
+	tmake_file="${tmake_file} arm/t-arm arm/t-linux arm/t-arm-elf"
 	extra_parts="crtbegin.o crtbeginS.o crtend.o crtendS.o"
 	gnu_ld=yes
 	;;
